test/test_jit_cuda_fuser.py

```diff
@@ -4,14 +4,20 @@
 
 import torch
 
-from torch.testing._internal.common_utils import run_tests, ProfilingMode, GRAPH_EXECUTOR
+from torch.testing._internal.common_utils import run_tests, ProfilingMode, GRAPH_EXECUTOR, UNITTEST_ARGS
 from torch.testing._internal.codegen.random_topo_test import runDefaultTestWithSeed
 
 from test_jit import JitTestCase, RUN_CUDA
 import itertools
 import numpy as np
 
-os.environ['PYTORCH_NVFUSER_DISABLE_FALLBACK'] = '1'
+if '-nvfuser_do_fallback' in UNITTEST_ARGS:
+    NVFUSER_DISABLE_FALLBACK = False
```

Comment:
Actually quite a few of our tests fail with fallback enabled. Seems like the errors were in the jit frontend (The upstream one). Should we just 

- Skip those that failed when fallback enabled (for sure skipping only when fallback enabled )or
- Actually have a separate test file for fallback enabled or
- Try to pin down those errors and fix them ?